src/ashwini/A20_SumOfChar.java

```diff
@@ -0,0 +1,27 @@
+package ashwini;
+
+public class A20_SumOfChar {
+
+	int digitCount = 0;
+	int sum = 0;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.